101-vm-with-data-management-gateway/README.md

```diff
@@ -0,0 +1,34 @@
+# Create a data management gateway and make it workable in azure VM
+
+Before deploying the template you must have the following
+
+1. **Data Factory.** The gateway is created in the data factory. If you don't have a data factory,  see the [Create data factory](https://docs.microsoft.com/en-us/azure/data-factory/data-factory-move-data-between-onprem-and-cloud#create-data-factory) for steps to create one.
```

Comment:
do you mean paste a quickstart sample link here? In this template(https://github.com/Azure/azure-quickstart-templates/tree/master/101-data-factory-blob-to-sql-copy-stored-proc),  it creates a data factory.